modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -3,28 +3,31 @@
 // * logging/cluster-logging-deploying.adoc
 
 [id="cluster-logging-deploy-multitenant_{context}"]
-= Installing OpenShift Logging into a multitenant network
+= Installing OpenShift Logging with SDN network isolation enabled
 
-If you are deploying OpenShift Logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+In a cluster, if the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#nw-openshift-sdn-modes_about-openshift-sdn[OpenShift SDN network isolation mode] uses the _Network policy_ or _multitenant_ modes, projects in that cluster are isolated from each other. Both of these modes prevent network traffic between pods or services that are in different projects. Network policy is the default mode in {product-title} {product-version}.
 
-Because the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
+When you deploy OpenShift Logging, the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects. Therefore, if a cluster is configured to use the Network policy or multitenant modes, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. The method you use to allow access depends on the network isolation mode.
```

Comment:
```suggestion
When you deploy OpenShift Logging, the OpenShift Elasticsearch Operator and the Red Hat OpenShift Logging Operator are installed in different projects. Therefore, if a cluster is using the OpenShift SDN cluster network provider configured to use the network policy or multitenant network isolation modes, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. The method you use to allow access depends on the network isolation mode.
```

Reply:
But even that isn't exactly correct; OpenShift SDN has these legacy network isolation modes. But OVN-Kubernetes is always in 'network policy' mode as it supports nothing else.

So you need to join projects for OpenShift SDN isolation mode multitenant.
And you need to create network policy objects for OVN-Kubernetes or OpenShift SDN isolation mode network policy.

I guess this is getting more complicated than one would hope for a small docs fix.

Reply:
Thanks for researching this and providing that extra information.

Reply:
I made significant changes that I hope reflect your suggestions.